src/markdown-pages/docs/managers/zos-manager.md

```diff
@@ -60,7 +60,52 @@ compileOnly 'dev.galasa:dev.galasa.zos.manager'
 # <a name="configuring"></a>Configuring 
 
 The following properties are used to configure the z/OS Manager.
- 
+
+To provision a CICS Region for your test you need to configure the following properties as a minimum to connect to a CICS region, even if you do not reference a `@ZosImage` in your Galasa test. This is because CICS Regions sit on a z/OS LPAR, and so to provision and connect to a CICS Region in a test, you also need access to the z/OS image that it sits within to make requests on the CICS Region. You might need to configure additional z/OS-related CPS properties, depending on your test.  
+
+
+```
+zos.dse.tag.[tag].imageid=[IMAGEID] OR zos.cluster.[clusterId].images=[IMAGEID]
+zos.image.[IMAGEID].ipv4.hostname=[IP ADDRESS]
+zos.image.[IMAGEID].credentials=[CREDENTIALID]
+```
+
+You also need to configure the following properties for the [CICS TS Manager](cics-ts-manager):
+
+```
+cicsts.provision.type=dse
+cicsts.dse.tag.[TAG].applid=[APPLID]
+```
```

Comment:
Perhaps we should add a sub-heading into the page before this section, something like 'Testing CICS Regions on z/OS' so it can be ignored for users who are not doing that, and it doesn't seem as though this is required for just z/OS tests.

Reply:
good idea - have updated